main.go

```diff
@@ -98,18 +98,32 @@ func (o *Options) LoadConfiguration(b []byte) error {
 	return nil
 }
 
-func run(ctx context.Context) error {
-	// Command line flags
-	var opt Options
-	opt.InitDefaults()
+func (o *Options) LoadConfigurationFile() error {
+	configPaths := []string{
+		"{CONFIG}/kubectl-ai/config.yaml",
```

Comment:
should we standardize on `${CONFIG}/.kubectl-ai` (hidden `kubectl-ai` ) ?

Reply:
It will be nice to share a sample config in the PR description. Can serve as documentation.